app/src/pages/review/[resourceType]/[id].tsx

```diff
@@ -34,16 +151,149 @@ export default function CommentPage() {
+                              <InfoCircle size="1rem" style={{ opacity: 0.5 }} />
+                            </div>
+                          </HoverCard.Target>
+                          <HoverCard.Dropdown>
+                            <Group>
+                              <Avatar color="blue" radius="xl">
+                                <InfoCircle size="1.5rem" />
+                              </Avatar>
+                              <Stack spacing={5}>
+                                <Text size="sm" weight={700} sx={{ lineHeight: 1 }}>
+                                  Learn More
+                                </Text>
+                                <Anchor
+                                  target="_blank"
+                                  href="https://build.fhir.org/ig/HL7/cqf-measures/ValueSet-artifact-comment-type.html"
+                                  color="dimmed"
+                                  size="xs"
+                                  sx={{ lineHeight: 1 }}
+                                >
+                                  Artifact Comment Types
+                                </Anchor>
+                              </Stack>
+                            </Group>
+                            <Space h="lg" />
+                            <List spacing="sm" size="sm" center>
+                              <List.Item>
+                                <i>Documentation:</i> The comment is providing additional documentation from an
+                                authoring perspective
+                              </List.Item>
+                              <List.Item>
+                                <i>Review:</i> The comment is providing feedback from a reviewer and requires resolution
+                              </List.Item>
+                              <List.Item>
+                                <i>Guidance:</i> The comment is providing usage guidance to an artifact consumer
+                              </List.Item>
+                            </List>
+                          </HoverCard.Dropdown>
+                        </HoverCard>
+                      </Group>
+                    }
+                    icon={<Star opacity={0.5} />}
+                    placeholder="Type"
+                    data={[
+                      { value: 'Documentation', label: 'Documentation' },
+                      { value: 'Guidance', label: 'Guidance' },
+                      { value: 'Review', label: 'Review' }
+                    ]}
+                    {...form.getInputProps('type')}
+                  />
+                </Group>
+                <Textarea
+                  radius="lg"
+                  mt="md"
+                  minRows={15}
+                  maxRows={15}
+                  placeholder="Your Artifact comment"
+                  label="Artifact Comment"
+                  description="Add a comment to the resource"
+                  withAsterisk
```

Comment:
It's a bit weird, but because I have the info icon beside the select, I can't include the withAsterisk property without the UI looking very...odd. I made it so that there is an asterisk, but  I did this by adding a red text mantine component that contains the *